### DISCUSSION TITLE
Github pages error! - 404 File Not Found :(

### DISCUSSION THREAD

**Discussion by ArAsHCoding** [community/community#59665] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi everyone,

I want to host my portfolio on Github pages but am getting the "404 file not found" error.
I have also tried most of the solutions to fix it. But still, the issue persists.

**Comment by fluentmoheshwar** [comment]
An index.html file must be present in root of the repository your index.html is in a folder called project. please move all files from project folder to root of the repository

**Reply by seo-asif** [reply]
To resolve the "404 file not found" error on GitHub Pages, you need to ensure that an `index.html` file is present in the root directory of your repository. Currently, your `index.html` file is located inside a folder called "project". To fix this, follow these steps:

1. Access your repository on GitHub.
2. Open the "project" folder in your repository.
3. Move all the files from the "project" folder to the root directory of your repository.
4. Once all the files are in the root directory, delete the now-empty "project" folder.
5. Commit and push the changes to your repository.

By moving the files from the "project" folder to the root directory, you ensure that the `index.html` file is in the correct location for GitHub Pages to find and serve it as the entry point for your portfolio website. After making these changes, your portfolio should be accessible without encountering the "404 file not found" error.

**Reply by ArAsHCoding** [reply]
Thanks!
It works now :)

**Reply by seo-asif** [reply]
Welcome dear buddy ❤️❤️... Follow me if you like or need help https://github.com/seo-asif

**Reply by GhostOf0days** [reply]
Just as advice, OP, I highly reccomend you stay away from Github Pages in the future and use something like Netlify or Vercel.
